### PR TITLE
Docker usage cleanup

### PR DESCRIPTION
**Pull request by glneo**
Some minor fixups to our usage of docker that I've come across while using this tool over the last couple days.

### CODE REVIEW

**Review by nmenon — COMMENTED** [review]
Lets document https://apt.llvm.org/llvm-snapshot.gpg.key in the commit log?

File: kpv [hunk 1]
```diff
@@ -55,7 +55,7 @@ if [[ $GIT_WORKTREE_COMMONDIR != ".git" ]]; then
 	DOCKER_MOUNT_DIRS+=" -v	$GIT_WORKTREE_COMMONDIR:$GIT_WORKTREE_COMMONDIR "
 fi
 
-export PATH=/workdir/scripts/dtc/:/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games:/opt/cross-gcc-linux-10/bin/:/opt/cross-gcc-linux-9/bin/
+export PATH=/workdir/scripts/dtc/:/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games:/opt/cross-gcc-linux-10/bin/:/opt/cross-gcc-linux-9/bin/:/usr/local/cross-gcc-linux-9/bin/
```
Comment (nmenon) [inline kpv:57]: Add /usr/local/cross-gcc-linux-10/bin/

File: kps [hunk 2]
```diff
@@ -53,7 +53,7 @@ if [[ $GIT_WORKTREE_COMMONDIR != ".git" ]]; then
 	DOCKER_MOUNT_DIRS+=" -v	$GIT_WORKTREE_COMMONDIR:$GIT_WORKTREE_COMMONDIR "
 fi
 
-export PATH=/workdir/scripts/dtc/:/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games:/opt/cross-gcc-linux-10/bin/:/opt/cross-gcc-linux-9/bin/
+export PATH=/workdir/scripts/dtc/:/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games:/opt/cross-gcc-linux-10/bin/:/opt/cross-gcc-linux-9/bin/:/usr/local/cross-gcc-linux-9/bin/
```
Comment (nmenon) [inline kps:48]: Add /usr/local/cross-gcc-linux-10/bin/

**Comment by nmenon** [comment]
MInor comments, but I think we are good to go.

**Comment by glneo** [comment]
Not sure what you mean by documenting the key, but I did add how one can download it themselves in the commit message.

**Pull request merged**